### PR TITLE
Minor UI tweaks for friendly URLs UI

### PR DESCRIPTION
**Pull request by Stanton**

### CODE REVIEW

File: stylesheets/_component.panels.scss [hunk 1]
```diff
@@ -15,6 +15,8 @@
 
     &__body {
         margin-left: 38px;
+        max-height: 20em;
+        overflow-y: auto;
```
Comment (langy) [inline stylesheets/_component.panels.scss]: Assuming autoprefixer will add --ms-overflow-y here for IE8 compat

**Comment by langy** [comment]
:+1:

File: stylesheets/_component.panels.scss (hunk 1 above)
Comment (Stanton) [inline stylesheets/_component.panels.scss]: I though that wasn't required for standards mode?

Comment (langy) [inline stylesheets/_component.panels.scss]: More Googling makes it seem that way, I felt sure overflow-x and -y had issues in one browser though, maybe it was Firefox and it's no longer a problem.

Unrelated issue with IE8 and combining overflow auto with max-height or max-width though: http://stackoverflow.com/questions/7707/ie8-overflowauto-with-max-height

**Pull request merged**